### DISCUSSION TITLE
MCS%40Rice/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#94] [Announcements]
# MCS%40Rice/

https://csmsapp.github.io/MCS%40Rice/

**Comment by syuan16** [comment]
rice 梯度这么低的吗

**Comment by NPgreatest** [comment]
24fall: class size 150人
小厂paycom只要认真准备基本可以拖底
整体找实习结果良好
可以称之为小找工技校

**Comment by MTDickens** [comment]
26 fall update：本科 zju，没交 GRE，gpa 3.7+（但是换算成美国计算方式只有 3.5），有两段非一作科研，Accept

所以 GRE 目前来讲可能并不非常看中（？

**Reply by lijiahuang1998** [reply]
同26fall！要不要➕ 微信 _129462谢谢
！

**Reply by Apolomooner** [reply]
我也26fall,可以加加联系方式

**Comment by lijiahuang1998** [comment]
Hi!
WeChat
_129462

**Comment by M1stF0rest** [comment]
美国找工数据很好，个人认为是性价比非常高的项目。
